app/src/main/java/com/infomaniak/mail/views/AvatarView.kt

```diff
@@ -138,29 +147,12 @@ class AvatarView @JvmOverloads constructor(
     }
 
     fun loadAvatar(correspondent: Correspondent?, bimi: Bimi? = null) {
-
-        fun loadSimpleCorrespondent(correspondent: Correspondent) {
-            loadAvatarUsingDictionary(correspondent, contacts = contactsFromViewModel)
-            currentCorrespondent = correspondent
-        }
-
         currentBimi = bimi
 
-        when (getAvatarDisplayType(correspondent, bimi)) {
-            AvatarDisplayType.UNKNOWN_CORRESPONDENT -> loadUnknownUserAvatar()
-            AvatarDisplayType.CUSTOM_AVATAR -> loadSimpleCorrespondent(correspondent!!)
-            AvatarDisplayType.BIMI -> loadBimiAvatar(ApiRoutes.bimi(bimi!!.svgContentUrl!!), correspondent!!)
-            AvatarDisplayType.INITIALS -> loadSimpleCorrespondent(correspondent!!)
-        }
-    }
+        val isBimiEnabled = avatarMergedContactData.isBimiEnabledLiveData.value ?: false
```

Comment:
This line is not accessed the same way `avatarMergedContactData.mergedContactLiveData.value` is. 

This access to `avatarMergedContactData.isBimiEnabledLiveData.value` should be made safe for when `isInEditMode` is true the same way we did it for the other live data